sdk/loadtestservice/loadtestservice/test/public/sample.jmx

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is this file?

Reply:
This file is a JMX file used to run a load test, this is a sample JMX file to run test upload file for load test.